developing-with-nats/tutorials/jwt.md

```diff
@@ -778,29 +777,29 @@ This will overwrite the stored account JWT.
 ##### Export Account - Non Operator/Administrator Environment - Self Service Deployment Modes
 
 This environment is set up with a signing key, thus the account is already [created properly signed](#createedit-account---all-environments---all-deployment-modes).
-The only step that is needed is to push the Account into the nats network.
+The only step that is needed is to push the Account into the NATS network.
 However, this depends on your ability to do so. 
 If you have no permissions, you have to perform the same steps as for the [decentralized deployment mode](#export-account---non-operatoradministrator-environment---decentralized-deployment-modes).
 The main difference is that upon import, the account won't be re-signed.
 
 #### Publicize an Account with Push - Operator Environment/Environment with push permissions - All Deployment Modes
 
 How accounts can be publicized wholly depends on the resolver you are using:
-* [mem-resolver](https://docs.nats.io/nats-server/configuration/securing_nats/jwt/resolver#memory): The operator has to have all accounts imported and generate a new config.
-* [url-resolver](https://docs.nats.io/nats-server/configuration/securing_nats/jwt/resolver#url-resolver): `nsc push` will send an HTTP POST request to the hosting webserver or `nats-account-server`.
+* [mem-resolver](../nats-server/configuration/securing_nats/jwt/resolver#memory): The operator has to have all accounts imported and generate a new config.
+* [url-resolver](../nats-server/configuration/securing_nats/jwt/resolver#url-resolver): `nsc push` will send an HTTP POST request to the hosting webserver or `nats-account-server`.
 * `nats-resolver`: Every environment with a system account user that has permissions to send properly signed account JWT as requests to:
   * `$SYS.REQ.CLAIMS.UPDATE` can upload and update all accounts. Currently, `nsc push` uses this subject.
   * `$SYS.REQ.ACCOUNT.*.CLAIMS.UPDATE` can upload and update specific accounts.
 
-`nsc generate config <resolver-type>` as a utility that generates the relevant nats config. 
+`nsc generate config <resolver-type>` as a utility that generates the relevant NATS config. 
 Where `<resolver-type>` can be `--mem-resolver` or `--nats-resolver` for the corresponding resolver.
-Typically the generated output is stored in a file that is then [included](link to doc) by the nats config.
-Every server within the same authentication domain needs to be configured with this configuration
+Typically the generated output is stored in a file that is then [included](link to doc) by the NATS config.
```

Comment:
Need to add missing 'link to doc'

Reply:
fixed by including link to [include directive](https://docs.nats.io/nats-server/configuration#include-directive)